evidence/alin/0005-alin-promote-II-2025.md

```diff
@@ -0,0 +1,177 @@
+like Mythos and Asset hub.
+
+On the relay chain, I implemented the [runtime 
+changes](https://github.com/paritytech/polkadot-sdk/pull/3479) necessary to support multiple backed 
+candidates per parachain. This lifted a core limitation and enabled parallel backing. I extended 
+these capabilities to the provisioner subsystem via 
+[PR 3233](https://github.com/paritytech/polkadot-sdk/pull/3233) and 
+[PR 3778](https://github.com/paritytech/polkadot-sdk/pull/3778), enabling nodes to handle multiple 
+candidates per para.
+
+To support out-of-order backing, I rewrote the prospective-parachains subsystem, 
+which was a critical bottleneck. This was a very complex endeavour that I owned.
+The resulting updates are captured in 
+[PR 4035](https://github.com/paritytech/polkadot-sdk/pull/4035) and 
+[PR 4937](https://github.com/paritytech/polkadot-sdk/pull/4937).
+
+Elastic scaling also required updates to Cumulus, including support in the PoV recovery component, 
+which I delivered in [PR 4733](https://github.com/paritytech/polkadot-sdk/pull/4733). To support 
+adoption, I authored a [comprehensive documentation 
+guide](https://github.com/paritytech/polkadot-sdk/pull/4663) for parachain teams.
+
+Getting the elastic scaling feature from the MVP to the production status involved two threads of work: RFC103 and
+deprecating AsyncBackingParams.
+
+[RFC103](https://github.com/polkadot-fellows/RFCs/pull/103) addresses a potential vulnerability 
+where a collator could distribute the same collation to all backing groups. I contributed
+numerous times to the node-side implementation (https://github.com/paritytech/polkadot-sdk/pull/6011, https://github.com/paritytech/polkadot-sdk/pull/5908)
+and owned the Cumulus-side support: https://github.com/paritytech/polkadot-sdk/pull/5372, https://github.com/paritytech/polkadot-sdk/pull/5888.
+
+To enable dynamic resource allocation, I [removed support for scheduling 
+TTL](https://github.com/paritytech/polkadot-sdk/pull/5461) from the coretime assignment pallet and 
+[deprecated `AsyncBackingParams`](https://github.com/paritytech/polkadot-sdk/pull/7254), which had 
+become obsolete with the introduction of the claim queue concept. Removing these static parameters 
+reduced validator overhead and improved spam resistance.
+
+In parallel, I participated in the design of [streamlined block 
+production](https://github.com/paritytech/polkadot-sdk/issues/5190#issuecomment-2505949587), proposing 
+a simplified post-state-based approach. I developed a [proof of 
+concept](https://github.com/paritytech/polkadot-sdk/tree/alindima/post-state-poc) and tested it using 
+gluttons and spammening parachains. While the need for this implementation hasn’t yet materialized, 
+the groundwork remains valuable for future scaling needs and showcases my ability to innovate and meaningfully
+contribute to areas outside of my previous expertise.
+
```

Comment:
I would add a very short paragraph at the end here describing the current path/plan for getting all of the above live (elastic scaling launch).

Reply:
I added a link to the issue that is tracking the launch (which contains the next steps). Listing them here seems out of scope and wouldn't age well either